modules/kubernetes-addons/main.tf

```diff
@@ -165,6 +165,22 @@ module "metrics_server" {
   addon_context     = local.addon_context
 }
 
+module "ondat" {
+  count             = var.enable_ondat ? 1 : 0
+  source            = "github.com/ondat/terraform-eksblueprints-ondat-addon"
```

Comment:
Could you please pin this to specific tag 
e.g.,   `version              = "0.0.1"`